model_hub/Makefile

```diff
@@ -17,10 +17,10 @@ TRANSFORMERS_VERSION := 4.8.2
 DATASETS_VERSION := 1.9.0
 TRANSFORMERS_ENVIRONMENT_ROOT := determinedai/model-hub-transformers
 # MMDetection Args
-MMCV_CUDA_VERSION := cu111 # Needs to match DEFAULT_GPU_IMAGE
-TORCH_VERSION := 1.9.0 # Needs to match DEFAULT_GPU_IMAGE
-MMCV_VERSION := 1.3.12
-MMDETECTION_VERSION := 2.16.0
+MMCV_CUDA_VERSION := cu113 # Needs to match DEFAULT_GPU_IMAGE
```

Comment:
This'll easily get forgotten. We should add a blurb about this to the bumpenvs README if, as I assume, this doesn't get updated automatically. Though I guess tests fail if you don't do it when you need it?

Reply:
yes, tests will fail. 

this is hardly a thing for bumpenvs - just like a support for new TF versions. I am not sure what kind of a README will be helpful for this.

Reply:
Just thinking that bumping CUDA versions and TORCH versions happens in a non-trivial portion of bumpenvs, even if not the majority. But as long as the tests do catch it, fair enough.